kolibri/core/assets/src/core_app.js

```diff
@@ -43,4 +43,4 @@ var Kolibri = function() {
     _.extend(this, _.pick(mediator, public_methods));
 };
 
-module.exports = Kolibri;
+module.exports = new Kolibri();
```

Comment:
Okay, that's fair enough. I took it out because it seemed like a singleton, but I hadn't considered unit testing. It's also a bit weird because we use this file in two ways: 

_kolibri_plugin.py:_

```python
class KolibriCoreFrontEnd(KolibriFrontEndPluginBase):
    """
    Plugin to handle
    """
    entry_file = "assets/src/core_app.js"
    external = True
    core = True
```

_kolibri_module.js:_

```javascript
const Kolibri = require('./core_app');
```

In the former, my understanding is the `exports` aren't used, and it may even instantiate a second copy when it hits the `require`?

I'll look into this.